10.9-libcxx/stable/main/finkinfo/libs/perlmods/string-diff-pm.info

```diff
@@ -1,19 +1,34 @@
 Info2: <<
 Package: string-diff-pm%type_pkg[perl]
-Version: 0.06
-Revision: 2
+Version: 0.07
+Revision: 1
 Depends: <<
 	algorithm-diff-pm,
-	perl%type_pkg[perl]-core
+	module-install-githubmeta-pm%type_pkg[perl],
```

Comment:
Sucks that such a simple pure-perl module has such a large and perlvarianted dependency chain. Are these all really Depends not BuildDepends? By grepping, I only see "Module::" in Makefile.PL and related places, not the library itself. I don't even see *anything* that is perlversion-related in the runtime (neither in the existing 0.06 version by comparison), just a TestDepends, so maybe it could become a build-varianted rather than runtime-varianted.

I can redo the packaging separately for that if you want to get the rest of this PR merged.

This package is Jesse Alama's...let's see if @jessealama is the right github account.

Reply:
module-install-githubmeta, M::I::readmefrompod, and M::I::readmemarkdownfrompod all Dep on module-install, which is varianted. Based on the deb filelist, I don't think they have anything that links/requires a specific perlversion, so they could conceivably be unvarianted, but would need a separate .info for 10.9.

Reply:
I'm not proposing unvarianting those packages--as you say, they have legitimate Depends on module-install-pmXXX, which is legitimately varianted. I'm just wondering about string-diff-pmXXX itself.

Reply:
@dmacks I'm pretty sure they're just BDeps.

Reply:
If all 4 M::I::* current Deps are really BDeps, then perl-core can also move to BDep. But that still leaves string-diff as a varianted package unless the usual systemperl 10.9 split is done, right?

Reply:
Right. ```Package:``` could be unvarianted and same ```Revision:``` on all platforms, but a systemperl *build* variant separate for 10.9.